src/elements/common/flowTypes.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+import { VIEW_MODE_GRID, VIEW_MODE_LIST } from '../../constants';
+
+export type ViewMode = typeof VIEW_MODE_GRID | typeof VIEW_MODE_LIST;
```

Comment:
Should this be in `/elements/common`? It seems to be used only by ContentExplorer at the moment.

Reply:
There has been some discussion if the gallery/grid view will ever make its way to the ContentPicker, so I put this in `/elements/common` for that reason.  Currently it is only being used in ContentExplorer, so does it make sense to move it to the `/elements/content-explorer` folder?

Reply:
Fair enough!